storage/innobase/include/dict0mem.h

```diff
@@ -1703,16 +1703,18 @@ struct dict_table_t {
 		return NULL;
 	}
 
-	/** Serialise metadata of dropped or reordered columns.
+	/** Serialise metadata BLOB, consisting of dropped or reordered columns,
+	and committed count.
 	@param[in,out]	heap	memory heap for allocation
 	@param[out]	field	data field with the metadata */
-	inline void serialise_columns(mem_heap_t* heap, dfield_t* field) const;
+	inline void serialise_mblob(mem_heap_t* heap, dfield_t* field);
```

Comment:
Why was the `const` attribute removed? The function comment does not say in which way `dict_table_t` would be modified.